yarn.lock

```diff
@@ -7882,6 +7882,11 @@ icss-utils@^4.1.0:
   dependencies:
     postcss "^7.0.14"
 
+idb-keyval@~3.2.0:
```

Comment:
All I needed was a persistent key value store, so I used [this](https://github.com/jakearchibald/idb-keyval)
It looks well maintained and popular enough. I pinned it to the minor version-  hopefully that's enough to keep things stable.

Reply:
sounds good